server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -641,6 +641,12 @@ public class UserVmManagerImpl extends ManagerBase implements UserVmManager, Vir
     private static final ConfigKey<Boolean> VmDestroyForcestop = new ConfigKey<Boolean>("Advanced", Boolean.class, "vm.destroy.forcestop", "false",
             "On destroy, force-stop takes this value ", true);
 
+    private static final ConfigKey<Integer> MigrateRetry = new ConfigKey<Integer>("Advanced", Integer.class, "migrate.retry", "10",
```

Comment:
Isn't 10 retries too many?
@benj-n do you plan to add any unit and integration tests for this?
IMO default behaviour should be no retries and then the operator can configure as desired

Reply:
Agreed. Default # of retries should be 1 (aka no retry).